session/handler.go

```diff
@@ -104,6 +111,114 @@ func (h *Handler) RegisterPublicRoutes(public *x.RouterPublic) {
 	public.DELETE(AdminRouteIdentitiesSessions, x.RedirectToAdminRoute(h.r))
 }
 
+// fandom-start
+// swagger:parameters adminIdentitySession
+//
+//nolint:deadcode,unused
+//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
+type adminIdentitySession struct {
+	// ID is the identity's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+
+	// Upgrade is a boolean flag that decides, if session should be upgrade to the highest possible level.
+	// If no value is provided, session returned is set with AAL1 level.
+	//
+	// required: false
+	// in: query
+	Upgrade bool `json:"upgrade"`
+}
+
+// swagger:model successfulAdminIdentitySession
+// nolint:deadcode,unused
+type AdminIdentitySessionResponse struct {
+	// The Session Token
+	//
+	// This field is only set when the session hook is configured as a post-registration hook.
+	//
+	// A session token is equivalent to a session cookie, but it can be sent in the HTTP Authorization
+	// Header:
+	//
+	// 		Authorization: bearer ${session-token}
+	//
+	// The session token is only issued for API flows, not for Browser flows!
+	Token string `json:"session_token"`
+
+	// Session
+	//
+	// The session contains information about the user, the session device, and so on.
+	//
+	// required: true
+	Session *Session `json:"session"`
+
+	// Identity
+	//
+	// The identity that just signed up.
+	//
+	// required: true
+	Identity *identity.Identity `json:"identity"`
+}
+
+// swagger:route GET /admin/identities/{id}/session identity adminIdentitySession
+//
+// Calling this endpoint issues a session for a given identity.
+//
+// This endpoint is useful for:
+//
+// - Issuing session or session token for a given identity without authenticating
+//
+//	Schemes: http, https
+//
+//	Security:
+//	  oryAccessToken:
+//
+//	Responses:
+//	  200: successfulAdminIdentitySession
+//	  404: errorGeneric
+//	  500: errorGeneric
+func (h *Handler) session(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	i, err := h.r.IdentityPool().GetIdentity(r.Context(), x.ParseUUID(ps.ByName("id")), identity.ExpandDefault)
```

Comment:
Suggestion: I think it could be `identity.ExpandNothing` to avoid extra joins for `VerifiableAddresses` and `RecoveryAddresses`

Reply:
Changed.